src/devices/cpu/z180/z180.h

```diff
@@ -107,10 +107,19 @@ class z180_device : public cpu_device, public z80_daisy_chain_interface
 public:
 	auto tend0_wr_callback() { return m_tend0_cb.bind(); }
 	auto tend1_wr_callback() { return m_tend1_cb.bind(); }
+	auto txa0_wr_callback() { return m_txa0_cb.bind(); }
+	auto txa1_wr_callback() { return m_txa1_cb.bind(); }
+	auto rts0_wr_callback() { return m_rts0_cb.bind(); }
```

Comment:
You don’t actually need trampolines, you can call `subdevice<...>(...).whatever()` here.